etsin_finder/frontend/js/components/qvain/history/TooltipContent.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+const TooltipContent = () => (
+  <>
+    <h2>some stuff</h2>
+    <div>some other stuff</div>
```

Comment:
Are these meant to use the `history.title` and `history.tooltip` translations?

Reply:
Not exactly. It's meant to use some translations in the future when there's proper content for tooltips. These texts being just placeholders. There's a good chance that content will be more complicated than just two different html elements so it would be no point to do any translations yet.